utils.py

```diff
@@ -48,6 +51,28 @@ def perform_rmd_conversion(stream):
             out_file.close()
             os.remove(out_path)
         return read_outfile
+    
+# We need to sanitize any code blocks (ex ```{bash}) from the rmd before rendering it
```

Comment:
I would move this comment into the sanitize_rmd function docstring, and add types:

def sanitize_rmd(data: String) -> bytes:
""" We need to sanitize any code blocks (ex ```{bash}) from the rmd before rendering it.
This is because kitr (https://rmarkdown.rstudio.com/authoring_quick_tour.html#Rendering_Output.), which powers our rendering actually executes any code in a labeled codeblock. By processing out any of these labels, we ensure we display a preview without a possible arbitrary code execution vulnerability 
See this document, issue number one, for more details on the vulnerability: https://docs.google.com/document/d/1aNCOKitTJH-GEkBSR4i-x91O0OQCZ8ZYa3feXtkja94/edit#heading=h.rvpr6zoz0jem
"""

Reply:
@jdcanas could you move the comments into the function docstring?